src/pages/useGapsList.ts

```diff
@@ -2,49 +2,24 @@ import { useEffect, useState } from 'react'
 import { Moment } from 'moment'
 import { getGapsAsync } from '../api/gapsService'
 
-import { sortByMode, HourlyData } from './components/utils'
+import { sortByMode, HourlyData, processData } from './components/utils'
 import { GapsList } from 'src/model/gaps'
 
-export const processData = (gapsList: GapsList): HourlyData[] => {
```

Comment:
when `processData` appears in the context of the `useGapsList` hook, it's easier to understand what it's purpose